src/material-experimental/mdc-slider/_slider-theme.scss

```diff
@@ -1,22 +1,40 @@
-// TODO: disabled until we implement the new MDC slider.
-// @import '@material/slider/mixins.import';
+@use '@material/slider/slider' as mdc-slider;
+@import '@material/slider/slider-theme';
 @import '../mdc-helpers/mdc-helpers';
+@import '../../material/core/ripple/ripple';
 
 @mixin mat-mdc-slider-color($config-or-theme) {
   $config: mat-get-color-config($config-or-theme);
   @include mat-using-mdc-theme($config) {
-    // TODO: disabled until we implement the new MDC slider.
-    // @include mdc-slider-core-styles($query: $mat-theme-styles-query);
+    @include mdc-slider.without-ripple($query: $mat-theme-styles-query);
 
     .mat-mdc-slider {
+      &.mat-primary, &.mat-accent, &.mat-warn {
+        $is-dark: map-get($config, is-dark);
+        $indicator-color: if($is-dark, white, black);
+        $indicator-text-color: if($is-dark, black, white);
+
+        @include value-indicator-color(
+          $color: $indicator-color,
+          $opacity: 0.6,
```

Comment:
I was thinking something like: `if($is-dark, $boosted-opacity, $original-opacity)`